Bugsnag/BugsnagCrashSentry.m

```diff
@@ -30,8 +30,7 @@ - (void)install:(BugsnagConfiguration *)config
     ksCrash.introspectMemory = YES;
     ksCrash.onCrash = onCrash;
     ksCrash.maxStoredReports = BSG_MAX_STORED_REPORTS;
-    ksCrash.threadTracingEnabled = (int) config.sendThreads;
-    
+
```

Comment:
It looks like default values are used inside KSCrash's `initWithReportFilesDirectory` and they are overridden with Bugsnag config values here.

Can the changes made to `BSG_KSCrash.m` therefore be made here instead?

```suggestion
ksCrash.threadTracingEnabled = config.sendThreads != BSGThreadSendPolicyNever;
```

Reply:
Updated to set `threadTracingEnabled` here